normandy/recipes/filters.py

```diff
@@ -353,6 +353,29 @@ def capabilities(self):
         return {"jexl.context.env.version", "jexl.transform.versionCompare"}
 
 
+class DateRangeFilter(BaseFilter):
+
+    type = "date_range"
+    not_before = serializers.DateTimeField()
+    not_after = serializers.DateTimeField()
+
+    def to_jexl(self):
+        not_before = self.initial_data["not_before"]
+        not_after = self.initial_data["not_after"]
+
+        return "&&".join(
+            [
+                f'(normandy.request_time>"{not_before}"|date)',
+                f'(normandy.request_time<"{not_after}"|date)',
+            ]
+        )
+
+    @property
+    def capabilities(self):
+        # do we need to make sure 'date' is.. a capability? What is 'date'?
+        return set()
```

Comment:
The word for the thing that `date` is, is a JEXL transform. There is a capability that indicates that a client has the `date` transform, and that capability is `jexl.transform.date` (there isn't really a good way for you to have known that, sorry).